crates/bevy_gizmos/src/lib.rs

```diff
@@ -146,6 +146,20 @@ pub trait AppGizmoBuilder {
     ///
     /// Configurations can be set using the [`GizmoConfigStore`] [`Resource`].
     fn init_gizmo_group<T: GizmoConfigGroup + Default>(&mut self) -> &mut Self;
+
+    /// Insert the [`GizmoConfigGroup`] in the app with the given value and [`GizmoConfig`].
+    ///
+    /// This might be useful for groups with fields configurations.
+    fn insert_gizmo_group<T: GizmoConfigGroup>(
+        &mut self,
+        group: T,
+        config: GizmoConfig,
+    ) -> &mut Self;
+
+    /// Insert the [`GizmoConfig`] in the given [`GizmoConfigGroup`]
+    ///
+    /// This method expects the given group to be already inserted in [`GizmoConfigStore`].
+    fn set_gizmo_group_config<T: GizmoConfigGroup>(&mut self, config: GizmoConfig) -> &mut Self;
```

Comment:
This method doesn't seem useful on `App`. If you have an `App`, you're almost certainly before app initialization and can just set it up correctly in the first place.

Otherwise, I would suggest just sticking to the resource access for mutating settings.

Reply:
Pretty fair, if the user needs to change a config of an already inserted config, they should just use the world inside the App. I'll remove this.